azure-kusto-data/azure/kusto/data/_token_providers.py

```diff
@@ -307,7 +308,7 @@ def _init_impl(self):
     def _get_token_impl(self) -> dict:
         try:
             if self._msi_auth_context is None:
-                self._msi_auth_context = ManagedIdentityCredential(**self._msi_args)
+                self._msi_auth_context = ManagedIdentityCredential(**self._msi_args, proxies=self._proxies)

```

Comment:
That does not make sense.
MSI is A) Always in Azure, B) Always on a link local IP for your current node.
Can't say if this is redundant or a security issue, but let's not allow this